src/sentry/plugins/providers/repository.py

```diff
@@ -83,3 +83,7 @@ def delete_repository(self, repo, actor=None):
 
     def compare_commits(self, repo, start_sha, end_sha, actor=None):
         raise NotImplementedError
+
+    @staticmethod
+    def ignore_commit(message):
+        return 'SKIP-SENTRY' in message
```

Comment:
This is a really weird indicator. Can we just do `#skipsentry`? This follows along with other conventions. `#nochanges` and `#skipci` etc.